_v2_source/index.html

```diff
@@ -0,0 +1,162 @@
+﻿<!DOCTYPE HTML>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Wedding Dashboard</title>
+    <style>
+        body {
+            margin: 0;
+            padding: 10px;
+            background: grey;
+            color: white;
+            font-family: Tahoma, Geneva, sans-serif;
+            font-size: 24px;
+        }
+
+        .splitflap {
+            margin: 0 auto;
+            -webkit-perspective-origin: top;
+            -moz-perspective-origin: top;
+            -ms-perspective-origin: top;
+            perspective-origin: top;
+            -webkit-perspective: 900px;
+            -moz-perspective: 900px;
+            -ms-perspective: 900px;
+            perspective: 900px;
+        }
+
+        #dashboard {
+            display: block;
+            width: 100%;
+            margin: 0 auto;
+            overflow: hidden;
+        }
+
+        #firstSection {
+            float: left;
+        }
+
+        #secondSection {
+            margin-left: 5px;
+            float: left;
+        }
+
+        #thirdSection {
+            margin-left: 5px;
+            float: left;
+        }
+    </style>
+    <script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
+    <script src="js/guests.js"></script>
+    <script src="js/ordering.js"></script>
+    <script src="js/jquery.splitflap.js"></script>
+</head>
+<body>
+    <div class="heading"></div> <br />
+    <div id="dashboard">
+        <div id="firstSection"></div>
+        <div id="secondSection"></div>
+        <div id="thirdSection"></div>
+    </div>
+
+    <script>
+        (function ($) {
+            $(document).ready(function () {
+                var guests = getGuests();
+                var ordering = getOrdering(guests.length);
+
+                initHeading($);
+                initDashboard($, guests);
+
+                window.setTimeout(function () {
+                    animateDashboard($, ordering, guests.length, 0);
+                }, 2000);
```

Comment:
It looks nicer and more readable to create CONSTANT_VARIABLE for these timing valures